src/components/config_profile/test/profile_test.cc

```diff
@@ -201,10 +201,12 @@ TEST_F(ProfileTest, UpdateBoolValues) {
 TEST_F(ProfileTest, UpdateStringValue) {
   // Default values
   std::string recording_file_name = "record.wav";
+  std::string server_address = "127.0.0.1";
+  std::string app_resource_folder = "";
   std::string tts_delimiter_ = "";
   std::string vr_help_title_ = "";
-  std::string app_resource_folder = "";
-  std::string server_address = "127.0.0.1";
+  std::string menu_title = "";
+  std::string menu_icon = "";
 
```

Comment:
@dcherniev By default `std::string` is constructed empty, so there is no need to assign `""`